src/components/ebay-pagination/examples/1-basic/template.marko

```diff
@@ -0,0 +1,13 @@
+<ebay-pagination aria-label-prev="Previous page" aria-label-next="Next page" curr-text="Results Pagination - Page 1">
```

Comment:
In the carousel, I omitted a11y attributes on the examples, except for one example that holds all of them: https://github.com/eBay/ebayui-core/pull/81/files#diff-b6c1a03d5e16301d2315f34a8b093c47R13

This way we can showcase the simplest usage (since all the a11y strings have defaults).

Reply:
Okay will update these.